### ISSUE TITLE
Check for rc / beta / alpha

### ISSUE THREAD

**Issue by jeremejevs** [dylang/npm-check#174]
This wouldn't be a good default, of course, but would be nice to be able to check for new[-er] rc / beta / alpha releases. Webpack 2, for example, has been in beta for 9 months now, with a fresh release every week or so, and I'd like to keep up with the same ease as I'm able to with regular releases.

**Comment by LinusU** [comment]
I agree, nice feature to have. Pull requests welcome 👍

**Comment by vith** [comment]
> This wouldn't be a good default

I disagree, I think this absolutely *should* be default for dependencies whose installed versions are beta/etc.

At present, `npm-check` is silent about my babel deps. `npm outdated` shows them:

```
❯ npm outdated
Package                                           Current          Wanted         Latest
@babel/cli                                  7.0.0-beta.46   7.0.0-beta.49  7.0.0-beta.49
@babel/core                                 7.0.0-beta.46   7.0.0-beta.49  7.0.0-beta.49
@babel/plugin-proposal-object-rest-spread   7.0.0-beta.46   7.0.0-beta.49  7.0.0-beta.49
@babel/preset-env                           7.0.0-beta.46   7.0.0-beta.49  7.0.0-beta.49
```

and a `yarn upgrade` upgraded them.

But I was in the habit of only checking with `npm-check` so I overlooked these upgrades for a long time.

**Comment by jeremejevs** [comment]
I'm fully satisfied with `yarn upgrade-interactive --latest` these days.
